flink-table/flink-table-common/src/main/java/org/apache/flink/table/functions/BuiltInFunctionDefinition.java

```diff
@@ -292,11 +293,18 @@ public Builder kind(FunctionKind kind) {
             return this;
         }
 
+        public Builder staticArguments(StaticArgument... staticArguments) {
+            this.typeInferenceBuilder.staticArguments(staticArguments);
+            return this;
+        }
+
+        /** @deprecated Use {@link #staticArguments(StaticArgument...)} instead. */
```

Comment:
Shouldn't we add `@Deprecated` annotation as well?
At least this is what javadoc for `@Deprecated` says
>The @Deprecated annotation should always be present if the @deprecated javadoc tag is present, and vice-versa.

Reply:
Of course. I missed this.